### ISSUE TITLE
Wrong sorting for S03E01 vs S02E01

### ISSUE THREAD

**Issue by klesun** [klesun/kunkka-torrent#2]
```
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E01 - The Boy in the Iceberg.mkv | 288.170 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E02 - The Avatar Returns.mkv | 269.373 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E03 - The Southern Air Temple.mkv | 288.675 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E04 - The Warriors of Kyoshi.mkv | 307.550 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E05 - The King of Omashu.mkv | 341.151 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E06 - Imprisoned.mkv | 269.638 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E07 - Winter Solstice, Part 1 - The Spirit World.mkv | 294.466 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E08 - Winter Solstice, Part 2 - Avatar Roku.mkv | 280.691 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E09 - The Waterbending Scroll.mkv | 281.604 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E10 - Jet.mkv | 284.704 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E11 - The Great Divide.mkv | 290.330 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E12 - The Storm.mkv | 338.828 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E13 - The Blue Spirit.mkv | 212.550 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E14 - The Fortuneteller.mkv | 290.290 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E15 - Bato of the Water Tribe.mkv | 306.392 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E16 - The Deserter.mkv | 275.301 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E17 - The Northern Air Temple.mkv | 268.866 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E18 - The Waterbending Master.mkv | 235.338 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E19 - The Siege of the North, Part 1.mkv | 253.596 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book One - Water/Avatar - The Last Airbender - S01E20 - The Siege of the North, Part 2.mkv | 238.570 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E01 - The Awakening.mkv | 252.608 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E02 - The Headband.mkv | 217.099 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E03 - The Painted Lady.mkv | 230.810 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E04 - Sokka's Master.mkv | 246.832 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E05 - The Beach.mkv | 236.539 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E06 - The Avatar and the Firelord.mkv | 267.107 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E07 - The Runaway.mkv | 259.045 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E08 - The Puppetmaster.mkv | 235.677 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E09 - Nightmares and Daydreams.mkv | 245.954 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E10&E11 - The Day of Black Sun, Part 1 - The Invasion, and Part 2 - The Eclipse.mkv | 514.288 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E12 - The Western Air Temple.mkv | 233.015 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E13 - The Firebending Masters.mkv | 265.056 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E14&E15 - The Boiling Rock, Part 1 and Part 2.mkv | 422.626 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E16 - The Southern Raiders.mkv | 294.530 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E17 - The Ember Island Players.mkv | 231.480 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Three - Fire/Avatar - The Last Airbender - S03E18-E21 - Sozin's Comet, Part 1 - The Phoenix King, Part 2 - The Old Masters, Part 3 - Into the Inferno, and Part 4 - Avatar Aang.mkv | 1026.178 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E01 - The Avatar State.mkv | 271.511 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E02 - The Cave of Two Lovers.mkv | 268.864 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E03 - Return to Omashu.mkv | 271.670 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E04 - The Swamp.mkv | 251.874 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E05 - Avatar Day.mkv | 294.757 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E06 - The Blind Bandit.mkv | 258.369 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E07 - Zuko Alone.mkv | 233.183 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E08 - The Chase.mkv | 274.659 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E09 - Bitter Work.mkv | 270.147 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E10 - The Library.mkv | 246.378 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E11 - The Desert.mkv | 256.671 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E12&E13 - The Serpent's Pass, and The Drill.mkv | 496.440 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E14 - City of Walls and Secrets.mkv | 202.140 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E15 - The Tales of Ba Sing Se.mkv | 267.035 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E16 - Appa's Lost Days.mkv | 269.851 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E17 - Lake Laogai.mkv | 232.352 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E18 - The Earth King.mkv | 243.307 MiB | Watch
Avatar - The Last Airbender (2005 - 2008) [1080p]/Book Two - Earth/Avatar - The Last Airbender - S02E19&E20 - The Guru, and The Crossroads of Destiny.mkv
```

**Comment by klesun** [comment]
Oh, did not notice words One/Three/Two... I wonder if they could be sorted still...

**Comment by klesun** [comment]
Fixed in 10680740ad5c31109cba50850d7c7027089e773c
